lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -24,7 +24,7 @@
 LOG = logging.getLogger(__name__)
 
 
-REVISION = 7
+REVISION = 8
```

Comment:
I think this revision change belongs in the previous commit.

Reply:
Ah, yes. Thank you. Also need to bump it up to 9.